scripts/utils/dateUtils.ts

```diff
@@ -0,0 +1,57 @@
+import { testVariadicArgs } from './cliUtils'
+
+export const isDateString = (date: string): boolean => {
+  const dateFormatRegex = /^\d{4}-\d{2}-\d{2}$/
```

Comment:
Nitpick: it's good practice to place this in a constants.ts file and then use it in here and the tests